applications/rag/metadata.yaml

```diff
@@ -97,7 +97,7 @@ spec:
       - name: frontend_domain
         description: Domain used for application and SSL certificate. 
         varType: string
-        defaultValue: "rag.example.com"
+        defaultValue: "frontend.example.com"
```

Comment:
Is this the default value for the terraform templates? May need to be updated to "".